src/Adapter/ParameterContainer.php

```diff
@@ -160,6 +173,20 @@ public function offsetUnset($name)
     public function setFromArray(array $data)
     {
         foreach ($data as $n => $v) {
+            if (is_string($n)) {
+                if (! isset($this->nameMapping[$n])) {
+                    foreach ($this->data as $nm => $vm) {
+                        if (ltrim(':', $vm) == ltrim(':', $n)) {
```

Comment:
```suggestion
                        if (ltrim($vm, ':') == ltrim($n, ':')) {
```
https://www.php.net/manual/en/function.ltrim.php